samba/umbrel-app.yml

```
@@ -0,0 +1,26 @@
+manifestVersion: 1
+id: samba
+name: Samba
+tagline: Make your storage accessible using Samba
+category: Networking
+version: "4.20.1"
+port: 9445
+description: >-
+  Samba is the standard Windows interoperability suite of programs for Linux and Unix.
+  Samba is Free Software licensed under the GNU General Public License, the Samba project is a member of the Software Freedom Conservancy.
+  Since 1992, Samba has provided secure, stable and fast file and print services for all clients using the SMB/CIFS protocol, such as all versions of DOS and Windows, OS/2, Linux and many others.
+  Samba is an important component to seamlessly integrate Linux/Unix Servers and Desktops into Active Directory environments. It can function both as a domain controller or as a regular domain member. 
+
+developer: Samba
+website: https://www.samba.org/
+submitter: João Victor
```

Add the permission to access the umbrelOS Downloads folder
```suggestion
submitter: João Victor
permissions:
  - STORAGE_DOWNLOADS
```